<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MySettingWidget</class>
 <widget class="QWidget" name="MySettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QToolBox" name="toolBox">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="page_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>242</width>
            <height>288</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Page 1</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="page_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>242</width>
            <height>288</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Page 2</string>
          </attribute>
          <widget class="QDateEdit" name="dateEdit">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>60</y>
             <width>110</width>
             <height>24</height>
            </rect>
           </property>
           <property name="displayFormat">
            <string>yyyy/MM/dd</string>
           </property>
          </widget>
          <widget class="QDateEdit" name="dateEdit_2">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>100</y>
             <width>110</width>
             <height>24</height>
            </rect>
           </property>
          </widget>
          <widget class="QDateTimeEdit" name="dateTimeEdit">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>160</y>
             <width>194</width>
             <height>24</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTableView" name="tableView"/>
       </item>
       <item row="1" column="0">
        <widget class="QFontComboBox" name="fontComboBox"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
